### PR TITLE
Ported Polaris/nearlyNon's Cards Against The Galaxy commit.

### PR DESCRIPTION
**Pull request by MistakeNot4892**
Copied strings, icons and some code from https://github.com/Baystation12/Baystation12/pull/18342 which was a port of my Polaris PR.
CAH is licensed under the same CC as our icons more or less so I don't thiiiink there are any licensing problems with including a game with the same format.

### CODE REVIEW

**Comment by MistakeNot4892** [comment]
Swapped `copytext_char` out for `copytext` for the time being.

File: config/example/cag_card_text_white.txt [hunk 1]
```diff
@@ -0,0 +1,147 @@
+Those motherfucking carp
+Having sex in the maintenance tunnels
+Space 'Nam
+Medical-Grade Lesbians
+The Gardener getting SUPER high
+The CO thinking they're a badass
+Being in a cult
+Racially biased lawsets
+An Unathi who WON'T STOP FIGHTING
+Tajara fetishists
```
Comment (afterthought2) [inline config/example/cag_card_text_white.txt:10]: Tajara and Unathi don't exist on stock neb.

File: config/example/cag_card_text_white.txt [hunk 2]
```diff
@@ -0,0 +1,147 @@
+Those motherfucking carp
+Having sex in the maintenance tunnels
+Space 'Nam
+Medical-Grade Lesbians
+The Gardener getting SUPER high
+The CO thinking they're a badass
```
Comment (afterthought2) [inline config/example/cag_card_text_white.txt:6]: CO deosn't exist, I don't think.

File: config/example/cag_card_text_white.txt [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+Forehead-mounted laser weaponry
+Pelvis-mounted laser weaponry
+Cardboard cutouts
+An obscene amount of rubber ducks
+Brain cakes
+A drone's game of life.
+Anomaly suits
+A gardener smoking reishi
+Military-grade baseball bats
+Angry SEA
```
Comment (afterthought2) [inline config/example/cag_card_text_white.txt:53]: This doesn't exist.

File: config/example/cag_card_text_white.txt [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+Trying to stab someone and hugging them instead
+Waking up naked in the maintenance tunnels
+Horrible cloning accidents
+Licking the supermatter on a dare
+A Deck Officer who WON'T STOP ordering guns
+Teaching a synthetic the birds and the bees
+Unnecessary surgery
+My addiction to spiders
+21 layer sandwiches
+wetskrell.nt
```
Comment (afterthought2) [inline config/example/cag_card_text_white.txt:88]: skrell reference

File: config/example/cag_card_text_white.txt [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+An irritatingly chipper robot
+Androids hanging out in the bar drinking beer
+Gear harnesses
+A seventeen-year-old CO
+The throbbing erection that the CoS gets at the thought of shooting something
+Trying to stab someone and hugging them instead
+Waking up naked in the maintenance tunnels
+Horrible cloning accidents
+Licking the supermatter on a dare
+A Deck Officer who WON'T STOP ordering guns
```
Comment (afterthought2) [inline config/example/cag_card_text_white.txt:83]: Job doesn't exist.

File: config/example/cag_card_text_white.txt [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+Innocent 16 year old stowaways
+A group of Skrell getting their squish on
+Enough soporific to put the entire ship down
+Delicious Vietnamese cuisine
+Wacky IPCs
+Spaceball Jones
+Jack Trasen
+Overreaching Nanotrasen Asset Protection
+The RD getting murdered horribly during an away mission
+Vox stowaways
```
Comment (afterthought2) [inline config/example/cag_card_text_white.txt:98]: I don't recall if we have Vox.

File: config/example/cag_card_text_white.txt [hunk 7]
```diff
@@ -0,0 +1,147 @@
...
+Horrible cloning accidents
+Licking the supermatter on a dare
+A Deck Officer who WON'T STOP ordering guns
+Teaching a synthetic the birds and the bees
+Unnecessary surgery
+My addiction to spiders
+21 layer sandwiches
+wetskrell.nt
+Innocent 16 year old stowaways
+A group of Skrell getting their squish on
```
Comment (afterthought2) [inline config/example/cag_card_text_white.txt:90]: More skrell.

File: config/example/cag_card_text_white.txt [hunk 8]
```diff
@@ -0,0 +1,147 @@
...
+Enough soporific to put the entire ship down
+Delicious Vietnamese cuisine
+Wacky IPCs
+Spaceball Jones
+Jack Trasen
+Overreaching Nanotrasen Asset Protection
+The RD getting murdered horribly during an away mission
+Vox stowaways
+Dead Resomii
```
Comment (afterthought2) [inline config/example/cag_card_text_white.txt:99]: Don't exist.

File: config/example/cag_card_text_white.txt [hunk 9]
```diff
@@ -0,0 +1,147 @@
...
+Diona nymphs in the vending machines
+Egg-laying
+Rouge AI
+Straightjacket and sedation tactics
+Collectable figurines
+Honkers the mentally unstable clown
+Free Trade Union protestors
+Underwear
+Civilian Contractors
+An armed marine XO
```
Comment (afterthought2) [inline config/example/cag_card_text_white.txt:109]: Role doesn't exist.

File: config/example/cag_card_text_white.txt [hunk 10]
```diff
@@ -0,0 +1,147 @@
...
+Twenty units of polytrinic acid
+Putting Ian in the washing machine
+An engineer with a burned hand, welder's eye, and a busted lung
+Runtime's mouse hunting ability
+Killing the cook and serving them to the crew as burgers
+Running a drug racket out of Cargo
+Three bottles of vodka
+Vox underarm stench
+The maintenance drone that saved the ship
+A Dylovene overdose, somehow
```
Comment (afterthought2) [inline config/example/cag_card_text_white.txt:124]: Improper chemical name.

File: config/example/cag_card_text_white.txt [hunk 11]
```diff
@@ -0,0 +1,147 @@
...
+Non-regulation hairstyles
+Floor-length braids
+The only straight woman aboard
+Flooding the ship with nitrous oxide
+Maniacal laughter and mustache-twirling
+Multiple organ failure
+Skrell intellectual superiority
```
Comment (afterthought2) [inline config/example/cag_card_text_white.txt:131]: More skrell.

File: config/example/cag_card_text_white.txt [hunk 12]
```diff
@@ -0,0 +1,147 @@
...
+Giving up on negotiation and just shooting them instead
+Shooting the hostage
+A printed and bound collection of stowaways' sob stories
+Medical malpractice
+A custom engine setup
+Diona Roast
+Robusting them with a toolbox
+Hiding the body in a locker, wrapping the locker in paper, and sending it through disposals
+Scientifically investigating how much hooch you can drink before passing out
+A Skrell who eats nothing but hallucinogenic mushrooms
```
Comment (afterthought2) [inline config/example/cag_card_text_white.txt:147]: Even more skrell.

**Review by afterthought2 — APPROVED** [review]

**Pull request merged**